app/components/emoji_picker/emoji_picker.js

```diff
@@ -67,11 +76,17 @@ class EmojiPicker extends PureComponent {
             emojis: this.props.emojis,
             searchTerm: ''
         });
-    }
+    };
 
     filterEmojiAliases = (aliases, searchTerm) => {
         return aliases.findIndex((alias) => alias.includes(searchTerm)) !== -1;
-    }
+    };
+
+    orientationDidChange = () => {
+        setTimeout(() => {
```

Comment:
Why the timeout here and in the other listeners? This could cause problems with the component being unmounted before the state change is triggered

Reply:
we need to give time to the orientation ends so that the new dimensions are set, I'll add a mounted check to prevent it to trigger the state if the components are unmounted